spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,16 @@
       end
     end
   end
+
+  describe 'POST create' do
+    context 'unauthenticated users' do
```

Comment:
Only a small note on "naming things"™: in order to end up with an almost human readable RSpec example, it makes sense to formulate contexts starting with "with" or "when", e.g. "with an unauthenticated user" or "without current_user".
Would be cool if you could come up with a naming like this, it in my opinion also increases readability :v: